aws/internal/service/networkfirewall/waiter/waiter.go

```diff
@@ -40,6 +40,7 @@ func FirewallUpdated(ctx context.Context, conn *networkfirewall.NetworkFirewall,
 		Target:  []string{networkfirewall.FirewallStatusValueReady},
 		Refresh: FirewallUpdatedStatus(ctx, conn, arn),
 		Timeout: FirewallTimeout,
+		Delay:   30 * time.Second,
```

Comment:
Added to account for Associate/DissociateSubnet calls that return a `Ready` status immediately after method is called instead of immediately returning `Provisioning`